UITests/TransferMethod/AddTransferMethodBankAccountTests.swift

```diff
@@ -20,21 +20,67 @@ class AddTransferMethodTests: BaseTests {
 
         waitForNonExistence(spinner)
 
-        addTransferMethod.setBranchId(branchId: "021000021")
-        addTransferMethod.setAccountNumber(accountNumber: "12345")
-        addTransferMethod.selectAccountType(accountType: "Checking")
+        addTransferMethod.setBranchId("021000021")
+        addTransferMethod.setAccountNumber("12346")
+        addTransferMethod.selectAccountType("Checking")
+        addTransferMethod.selectRelationship("Self")
+        addTransferMethod.setNameFirst("John")
+        addTransferMethod.setNameLast("Smith")
+        addTransferMethod.setNameMiddle("Adam")
+        addTransferMethod.setPhoneNumber("+16045555555")
+        addTransferMethod.setMobileNumber("+16046666666")
+        addTransferMethod.setDateOfBirth(yearOfBirth: "1981", monthOfBirth: "December", dayOfBirth: "25")
+        addTransferMethod.selectCountry("United States")
+        addTransferMethod.setStateProvince("Maine")
+        addTransferMethod.setStreet("632 Broadway")
+        addTransferMethod.setCity("Bangor")
+        addTransferMethod.setPostalCode("04401")
+
+        app.scrollToElement(element: addTransferMethod.createTransferMethodButton)
+
         addTransferMethod.clickCreateTransferMethodButton()
 
-        //Todo - check processing indicator
+        waitForNonExistence(spinner)
+
+        XCTAssert(app.navigationBars.staticTexts["Account Settings"].exists)
+    }
+
+    func testAddTransferMethod_createBankAccountBusiness() {
```

Comment:
You need to setup the test to return the graphql response for business fields, it currently fails when it tries to set the profile fields.  I would say that it is easier to create a new test class for this - AddTransferMethodBankAccountBusinessTests.

Reply:
@skoong Is there any manual/instruction how to set up the test properly? 
As neither me nor my team don't have clear vision on this yet.
-where to get/how to create auth data
-how to set up mock server to use these data
-where to get/how to create mock response data
-how to validate the response

Or, probably, you may point out any test that covers all the points listed above.
@bhordiienko-epam is working on ability to allow usage of various accounts, but doesn't seem to have much progress on this.

Reply:
Fixed